ext/http/00_serve.ts

```diff
@@ -581,6 +581,19 @@ type RawServeOptions = {
 
 const kLoadBalanced = Symbol("kLoadBalanced");
 
+function mapAnyAddrToLocalhostForWindows(hostname: string) {
+  // If the hostname is "0.0.0.0", we display "localhost" in console
+  // because browsers in Windows don't resolve "0.0.0.0".
+  // See the discussion in https://github.com/denoland/deno_std/issues/1165
+  if (
+    (hostname == "0.0.0.0" || hostname == "::") &&
+    (Deno.build.os === "windows")
```

Comment:
The evaluation won't change by reordering the conditions, but because the platform is a precondition of the others, I think it makes sense to go first (it will short-circuit more quickly for other platforms and avoid the unnecessary comparisons):

```suggestion
    (Deno.build.os === "windows") &&
    (hostname == "0.0.0.0" || hostname == "::")
```